RecoEgamma/EgammaTools/interface/GBRForestTools.h

```diff
@@ -1,39 +0,0 @@
-//--------------------------------------------------------------------------------------------------
-//
-// GRBForestTools
-//
-// Utility to read a TMVA weights file with a BDT into a GRBForest.
-//
-// Author: Jonas Rembser
-//--------------------------------------------------------------------------------------------------
-
-
-#ifndef RecoEgamma_EgammaTools_GBRForestTools_h
-#define RecoEgamma_EgammaTools_GBRForestTools_h
-
-#include <vector>
-#include <string>
-
-#include "CondFormats/EgammaObjects/interface/GBRForest.h"
-#include "FWCore/ParameterSet/interface/FileInPath.h"
-
-#include "TMVA/MethodBDT.h"
-#include "TMVA/Reader.h"
-
-#include "CommonTools/Utils/interface/TMVAZipReader.h"
-
-class GBRForestTools
-{
-  public:
-    GBRForestTools() {}
-
-    static std::unique_ptr<const GBRForest> createGBRForest(const std::string &weightFile);
-    static std::unique_ptr<const GBRForest> createGBRForest(const edm::FileInPath &weightFile);
```

Comment:
it seems to me that EgammaTools (not necessarily in RecoEgamma) is a more appropriate way to fill a GBRForest from non-DB sources, compared to doing this in CondFormats.
Perhaps even changing ```Egamma``` to something less specific is even better.
Maybe CommonTools/MVAUtils or CommonTools/BDTUtils